components/src/dynamo/trtllm/request_handlers/handler_base.py

```
@@ -107,13 +108,23 @@ def _collective_rpc(self, method: str, rpc_tags: list[str]) -> None:
-            # Some TRT-LLM versions use "wake_up" instead of "wakeup"
-            rpc("wake_up", args=(rpc_tags,), kwargs={}, non_block=False)
+            return
+
+        # Probe and cache the correct wakeup method name on the first call.
+        # Some TRT-LLM versions use "wake_up" instead of "wakeup".
+        if self._wakeup_rpc_name is None:
+            try:
+                rpc("wakeup", args=(rpc_tags,), kwargs={}, non_block=False)
+                self._wakeup_rpc_name = "wakeup"
+                return
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)